cmd/tendermint/commands/lite.go

```diff
@@ -39,6 +41,11 @@ need a primary RPC address, a trusted hash and height and witness RPC addresses
 (if not using sequential verification). To restart the node, thereafter
 only the chainID is required.
 
+When /abci_query is called, Cosmos SDK multistore Merkle key path format is used:
```

Comment:
```suggestion
When /abci_query is called, the Merkle key path format is used:
```
? Not sure if it makes sense this way or mentioning the SDK

Reply:
I second this: can't we spin it as a tendermint default key path that the cosmos sdk follows rather than the other way around

Reply:
yeah, we can rename to "ABCI default key path"